packages/forms-web-app/src/controllers/register/common/common-view-model.js

```diff
@@ -8,6 +8,7 @@ const pageTitle = {
 	[keys.organisation]: ` - Registering for an organisation ${commonPageTitle}`,
 	[keys.agent]: ` - Registering on behalf of someone else ${commonPageTitle}`
 };
+
```

Comment:
Nitpick comment: Would be good to revert this space so this file doesn't show as having changes when it only a rename